stabilityai/pytorch/inference/docker/1.13/py3/cu117/Dockerfile.gpu

```diff
@@ -0,0 +1,10 @@
+FROM 763104351884.dkr.ecr.us-west-2.amazonaws.com/pytorch-inference:1.13.1-gpu-py39-cu117-ubuntu20.04-sagemaker as sagemaker
+ARG SGM_VERSION=0.0.1
+
+# Install Stability Generative Models
+RUN cd /tmp \
+ && git clone https://github.com/stability-ai/generative-models -b palp/model-tests \
+ && cd generative-models \
+ && pip install -r requirements_pt13.txt \
+ && pip install . \
+ && rm -rf /tmp/generative-models
```

Comment:
can you add this block of code as well to add OSS compliance block? https://github.com/aws/deep-learning-containers/blob/master/pytorch/inference/docker/1.13/py3/cu117/Dockerfile.gpu#L306-L313